views/layouts/main.handlebars

```diff
@@ -7,8 +6,12 @@
   <title>The Sheet Show</title>
   <link rel="shortcut icon" href="./assets/logos/Logo.png">
   <link rel="stylesheet"    href="./assets/css/style.css">
+  <link rel="stylesheet" href="../assets/css/style.css">
   <link rel="stylesheet"    href="./assets/css/login.css">
+  
   <link rel="stylesheet"    href="./assets/css/navigation.css">
+  <link rel="stylesheet"    href="../assets/css/navigation.css">
+  <link rel="stylesheet" href="../assets/css/animations.css">
 
```

Comment:
Added pathing to CSS for a few scenarios knowing this is the wrong way to do it but just to save time in troubleshooting for now.